e2e/package.json

```diff
@@ -16,7 +16,7 @@
     "test:ci": "yarn _test --ci --maxWorkers=3 --forceExit",
     "test-local": "cross-env WORKBENCH_ENV=local DEBUG=true yarn _test --maxWorkers=3",
     "test-local:debug": "cross-env WORKBENCH_ENV=local PUPPETEER_DEBUG=true PUPPETEER_HEADLESS=false yarn _test --maxWorkers=1",
-    "test-nightly": "cross-env WORKBENCH_ENV=test TEST_MODE=nightly-integration yarn _test --maxWorkers=2",
+    "test-nightly": "cross-env WORKBENCH_ENV=test TEST_MODE=nightly-integration yarn _test --maxWorkers=3",
```

Comment:
please revert.  if you want to make this change, do it in isolation.

Reply:
Done